application/F3DOptionsParser.cxx

```diff
@@ -202,11 +221,12 @@ void ConfigurationOptions::GetOptions(F3DAppOptions& appOptions, f3d::options& o
     this->DeclareOption(grp0, "no-background", "", "No background when render to file", appOptions.NoBackground, HasDefault::YES, MayHaveConfig::YES);
     this->DeclareOption(grp0, "help", "h", "Print help");
     this->DeclareOption(grp0, "version", "", "Print version details");
-    this->DeclareOption(grp0, "readers-list", "", "Print the list of file types");
+    this->DeclareOption(grp0, "readers-list", "", "Print the list of readers");
```

Comment:
I think we need a --plugins-list, wdyt ?

Reply:
Could be useful indeed. I'll do it later.